### DISCUSSION TITLE
Better Documentation

### DISCUSSION THREAD

**Discussion by kaaax0815** [wg-easy/wg-easy#1112] [Ideas]
Better Documentation in terms of examples, screenshots and tutorials

I find the chart type feature especially undocumented because you dont know you have to click the chart button to active and dont know how each chart looks

Bar (3)
![firefox_75f1zxzjYX](https://github.com/wg-easy/wg-easy/assets/32197462/8d0a3bd6-a271-4584-87fc-6b06aeac8e9f)
Area (2)
![firefox_4BM0cDUlOx](https://github.com/wg-easy/wg-easy/assets/32197462/1744a5c8-0e83-4d17-907f-75eb6c47828e)
Line (1)
![firefox_vyUze61bQt](https://github.com/wg-easy/wg-easy/assets/32197462/cea55ccb-0e3a-48b4-9f2d-4757f7f060da)
None (0) hides the chart button in the top right corner

@pheiduck 
Posted here as there is no feature request issue. but unfortunate as discussions are not really read

**Comment by pheiduck** [comment]
@suxscribe are able to Document the feature?
